tests/unit/db/redshift/test_redshift_db_driver_unload.py

```diff
@@ -47,7 +47,8 @@ def test_unload_to_non_s3(self,
             'unload_location': self.mock_s3_temp_base_loc.temporary_directory().__enter__().url
         }
         mock_UnloadFromSelect.assert_called_with(**expected_args)
-        self.assertEqual(456, rows)
+        self.assertEqual('db_engine.connect().__enter__().execute().scalar()',
+                         rows._extract_mock_name())
```

Comment:
having a hard time understanding what this is doing, can you explain?

Reply:
This is now fixed